### ISSUE TITLE
filtering on sports ball

### ISSUE THREAD

**Issue by Jonas-DW** [immersive-limit/coco-manager#3]
i cant seem to filter on the category sports ball since it has a space in it and the code uses spacing for declaring multiple category

**Comment by akTwelve** [comment]
Yeah, I didn’t account for that unfortunately. The best workaround is to remove the space or replace it with an underscore and then remove it manually once the json is generated.

Adam Kelly

Sent from my mobile
________________________________

**Comment by Ekta246** [comment]
> Yeah, I didn’t account for that unfortunately. The best workaround is to remove the space or replace it with an underscore and then remove it manually once the json is generated. Adam Kelly
> […](#)
> Sent from my mobile
> ________________________________ From: Jonas-DW <notifications@github.com> Sent: Wednesday, November 18, 2020 8:47:32 AM To: immersive-limit/coco-manager <coco-manager@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [immersive-limit/coco-manager] filtering on sports ball (#3) i cant seem to filter on the category sports ball since it has a space in it and the code uses spacing for declaring multiple category — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fimmersive-limit%2Fcoco-manager%2Fissues%2F3&data=04%7C01%7C%7Cbdcaf2f64730435b59c908d88bd0e20b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637413076539834299%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=n5TtYAni7kjP%2FfttPYgq89MYm%2BWZ0qnI%2Bakv%2B02zn6A%3D&reserved=0>, or unsubscribe<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABUEBKMJYQP76UI35BENYRTSQPNAJANCNFSM4T2CCGSA&data=04%7C01%7C%7Cbdcaf2f64730435b59c908d88bd0e20b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637413076539844296%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=XqqbDPHiNFYM9BZe5kr9a6OY2Uzd%2FQRtzswZKwv34ug%3D&reserved=0>.

Isn't it quite tedious to do so?

**Comment by SuijkerbuijkP** [comment]
The easier workaround is to just add quotation marks around the category, e.g. "sports ball", which adds the entire string to the categories list.
